pkgs/servers/bazarr/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, lib, fetchurl, makeWrapper, python3, python3Packages, ... }:
+
+stdenv.mkDerivation rec {
+  pname = "bazarr";
+  version = "v0.8.4.3";
```

Comment:
Nix version-parsing (for `nix-env -u`) requires the version to start with a number.

Reply:
So just move that "v" out of the version attribute and prefix "v" before the version number where needed.